angular.json

```diff
@@ -62,7 +62,10 @@
             ],
             "styles": [
               "projects/ng-core-tester/src/styles.scss",
-              "node_modules/ngx-bootstrap/datepicker/bs-datepicker.css"
+              "node_modules/ngx-bootstrap/datepicker/bs-datepicker.css",
+              "node_modules/primeicons/primeicons.css",
+              "node_modules/primeng/resources/themes/lara-light-blue/theme.css",
```

Comment:
I'm a little bit confused about placing theming specification into ng-core.
Your component will use the `lara-light-blue` theme. But if customer project like "RER-ILS-UI" use a different theme, there will be an incoherence in the UX.
For me, `ng-core` should as less as possible including styling (into css, into component, ...) and let customer project to implement this theming. 

But it will work using this way. So it's not a "request changes"

Reply:
The styles have been added on the test project. It is not at the level of the main library.

Is it OK like this?

Reply:
@Garfield-fr I didn't notice this point. This is OK for me